### ISSUE TITLE
dwn unit tests fail intermittently once PR is merged into main

### ISSUE THREAD

**Issue by jiyoontbd** [TBD54566975/web5-js#477]
happened when https://github.com/TBD54566975/web5-js/pull/476 was merged. it was re-run and passed successfully, which points to the failing test being affected by race condition.

**Comment by LiranCohen** [comment]
Thanks @jiyoontbd!! I'll try to reproduce.

**Comment by jiyoontbd** [comment]
@LiranCohen ah! i found the ci run where it failed. i just had to click 5 levels deep to find it

https://github.com/TBD54566975/web5-js/actions/runs/8516766403/job/23326301339

**Comment by jiyoontbd** [comment]
maybe related - node-sdk-reports also fails intermittently https://github.com/TBD54566975/web5-js/actions/runs/8541288382/job/23400248626
